waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/mapping/model/ASTQueryMappingTest.java

```diff
@@ -34,7 +34,7 @@
   @Test
   public void transformOutboundDatabaseName() {
     ASTQueryMapping queryMapping = ASTQueryMapping.INSTANCE;
-    MetaStoreMapping metaStoreMapping = new MetaStoreMappingImpl(PREFIX, "mapping", null, null, DIRECT);
+    MetaStoreMapping metaStoreMapping = new MetaStoreMappingImpl(PREFIX, "mapping", null, null, DIRECT, 800l);
```

Comment:
Why 8001? Looks like a magic number to me....

Reply:
The number for timeout will change and will be different depending on the task executed (`setUgi`, `getTableMeta` or `getDatabases`). So, based on the graph that Patrick shared on Friday for WD tasks, we'll go with a higher limit and set that as a constant with a good, indicative name.